### DISCUSSION TITLE
Virtual desktop background images get mixed up

### DISCUSSION THREAD

**Discussion by alexander233** [valinet/ExplorerPatcher#2204] [General]
I am running ExplorerPatcher on windows 11. With the latest version I was able to use the Win+TAB key again to switch between virtual desktops. Thank you very much for this fix!

But now there is another problem: I am using three virtual desktops, each of which has its own background image, so that I can easier distinguish where I am right now. Unfortunately now these background images merge after switching a few times from virtual desktop to virtual desktop via CTRL-WIN-ARROW_LEFT or CTRL-WIN-ARROW_RIGHT. After a while all three desktops have the same background image (the one from the first desktop). 

I have tried several times to change the desktop images back. Initially they are different but then they become the same again...

Explorer Patcher Version: 22621.2361.58.1
Windows 11 Version: 22H2 22621.2283

**Comment by pyrates999** [comment]
Can you uninstall EP, reboot, and see if this still happens or not?

**Reply by alexander233** [reply]
It does not happen any more after uninstall of EP.

**Reply by pyrates999** [reply]
Ok, thank you.  You've now confirmed it and someone can look at it.
